core/train.go

```diff
@@ -740,6 +740,10 @@ func cancelTrain(r *http.Request) response {
 	messagingService := messaging.GetService()
 	messagingService.TrainCancelled(train, authedUser)
 
+	for job := range train.ActivePhases.Deploy.Jobs {
+		err = build.Jenkins().CancelJob(job.url)
```

Comment:
If we don't care about `err`, probably we can use `_`.
Otherwise we may want to handle error within the loop because `err` will be overwritten.

Reply:
It's almost never a good idea to ignore errors with `_`. It will eventually always come back to bite you unless you are sure it will never return an error. You will most always want to at least log it.